modules/apps/27-interchain-accounts/controller/types/msgs.go

```diff
@@ -85,3 +88,33 @@ func (msg MsgSendTx) GetSigners() []sdk.AccAddress {
 
 	return []sdk.AccAddress{accAddr}
 }
+
+// NewMsgUpdateParams creates a new MsgUpdateParams instance
+//
+//nolint:interfacer
```

Comment:
do you remember if this caused an issue? In general, seems like interfacer has been [deprecated](https://github.com/mvdan/interfacer). Unsure if this is a relic from the past?

Reply:
done